.warden/commands/bootstrap.cmd

```diff
@@ -18,15 +18,15 @@ cd "${WARDEN_ENV_PATH}"
 
 ## configure command defaults
 WARDEN_WEB_ROOT="$(echo "${WARDEN_WEB_ROOT:-/}" | sed 's#^/#./#')"
-REQUIRED_FILES=("${WARDEN_WEB_ROOT}/auth.json")
+REQUIRED_FILES=()
 DB_DUMP="${DB_DUMP:-./backfill/magento-db.sql.gz}"
 DB_IMPORT=1
 CLEAN_INSTALL=
 AUTO_PULL=1
 META_PACKAGE="magento/project-community-edition"
 META_VERSION=""
 URL_FRONT="https://${TRAEFIK_SUBDOMAIN}.${TRAEFIK_DOMAIN}/"
-URL_ADMIN="https://${TRAEFIK_SUBDOMAIN}.${TRAEFIK_DOMAIN}/backend/"
+URL_ADMIN="https://${TRAEFIK_SUBDOMAIN}.${TRAEFIK_DOMAIN}/admin/"
```

Comment:
Needs to be backend instead admin, due to INSTALL_FLAGS set --backend-frontname top 'backend'

Reply:
Fixed in the other location